idaes/core/util/model_statistics.py

```diff
@@ -377,7 +378,31 @@ def number_activated_equalities(block):
     Returns:
         Number of activated equality Constraint components in block
     """
-    return sum(1 for _ in activated_equalities_generator(block))
+    return sum(
+        1 for _ in activated_equalities_generator(block)
+    ) + number_grey_box_equalities(block)
+
+
+def number_grey_box_equalities(block):
+    """
+    Method to return the number of equality constraints in GreyBox
```

Comment:
I would say, "Compute total number of equality constraints for all GreyBox objects in this block."

Reply:
done